backend/app/api/routes/inquiries.py

```diff
@@ -24,3 +27,30 @@ def create_inquiry(*, session: SessionDep, inquiry_in: InquiryCreate) -> Inquiry
     session.commit()
     session.refresh(inquiry)
     return inquiry
+
+
+@router.get("/", response_model=InquriesPublic)
+def read_inquries(
+    session: SessionDep, skip: int = 0, limit: int = 100
```

Comment:
are skip and limit for pagination?

Reply:
Yes, it's for pagination. I implemented it in case we want to apply pagination for our inquiry table UI.

Reply:
cool.  thanks!